slam-libraries/viam-cartographer/scripts/rebuild_viam_cartographer.sh

```diff
@@ -13,6 +13,6 @@ fi
 
 # Rebuild viam cartographer.
 pushd build
+cmake .. -G Ninja -DCMAKE_CXX_STANDARD=17 -DCMAKE_PREFIX_PATH=`brew --prefix` -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5
```

Comment:
[q] Why did you add this line here? The idea of `rebuild_viam_cartographer.sh` is not having to run `cmake` again

Reply:
if new files were added then the rebuild script would miss them. running the cmake script does not take very long. The reason why `build_viam_cartographer` takes longer is because we delete the build directory and start from scratch every time